src/components/button/button-test.stories.tsx

```diff
@@ -1,25 +1,26 @@
-import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import React from "react";
 import { action } from "@storybook/addon-actions";
 
-import Button from ".";
+import Button, { ButtonProps } from ".";
+import Box from "../box";
 import { ICONS } from "../icon/icon-config";
 import {
   BUTTON_ICON_POSITIONS,
   BUTTON_SIZES,
   BUTTON_VARIANTS,
 } from "./button.config";
+import { ButtonIconPosition, ButtonTypes } from "./button.component";
 
-<Meta
-  title="Button/Test"
-  parameters={{
+export default {
+  title: "Button/Test",
+  excludeStories: ["ButtonDifferentTypes", "generateButtons"],
+  parameters: {
     info: { disable: true },
-    chromatic: {
-      disable: false,
-    },
-  }}
-/>
+    chromatic: { disable: true },
```

Comment:
have we decided to disable these stories in chromatic? I thought that we are testing every button combination in these stories.